### ISSUE TITLE
[Proposal] Add an additional APAC collector/agent SIG

### ISSUE THREAD

**Issue by MovieStoreGuy** [open-telemetry/opentelemetry-collector#7085]
Apologies if I have opened this in the wrong spot, please feel free move it into the correct place.

I am looking for an expression of interesting in running an additional meeting for those working in APAC and wanting to actively contribute back to the project.

Looking at the otel calendar,  4:00pm Pacific (Taken from the otel Instrumetation APAC SIG) works well for me. 

Or roughly -22:00 UTC through to +08:00 UTC (negative being the previous day respective to UTC)

**Comment by codeboten** [comment]
Discussed in the Feb 1 SIG call, the proposal is to add 2 meetings to the calendar:

- 1st Wednesday is EU-APAC friendly
- 3rd Wednesday is NA-APAC friendly

**Comment by mx-psi** [comment]
I am going to pin this for visibility, also cc @fatsheep9146 since this may interest you :)

**Comment by fatsheep9146** [comment]
@mx-psi @codeboten @MovieStoreGuy 

Thanks a lot! I truly have interest in this! This meeting is hold once a week or once a month?

**Comment by mx-psi** [comment]
AIUI we would have two more meetings a month that are APAC friendly, one happening in the morning and another one in the evening

**Comment by MovieStoreGuy** [comment]
With those proposed times, I am able to attend both the NA-APAC and EU-APAC.

Thank you @mx-psi for following up on this <3

**Comment by fatsheep9146** [comment]
Sorry for my late response @mx-psi 

As a APAC contributors, I think 07:00(UTC +08:00) for NA-APAC may be better, since the 08:00 is basically a commuting time for many peoples. For example, I basically go to companies in 8:10 - 8:30 since the company's regular bus is in that time.

**Comment by mx-psi** [comment]
> As a APAC contributors, I think 07:00(UTC +08:00) for NA-APAC may be better, since the 08:00 is basically a commuting time for many peoples. For example, I basically go to companies in 8:10 - 8:30 since the company's regular bus is in that time.

I assume this is about the NA-APAC time? So that would be:

|         | UTC−08:00 (PST) | UTC -05:00 (EST) | UTC +01:00 (CET) | UTC +08:00 (China) | UTC +10:00 (Sidney) |
|---------|-----------------|------------------|------------------|--------------------|---------------------|
| NA-APAC (alternative proposal) | **15:00**           | **18:00**            | *00:00*          | **07:00**              | **10:00**               |

I think that should be fine for everyone else, I worry that it's a bit too early for people in China/other countries using that TZ, but you are definitely the expert here so if it's fine for you it's fine for me too :smile:

**Comment by fatsheep9146** [comment]
Thanks a lot! This schedule is perfect for me! @mx-psi 
I think the 16:00 timeline is also suitable for other contributors.

**Comment by mtwo** [comment]
I've added both of these to the community calendar. Please let me know if there are any issues or if we need to edit the existing meetings (I left them untouched).

**Comment by mx-psi** [comment]
Thanks @mtwo! :heart:

**Comment by jpkrohling** [comment]
We are trying this again 🙂

**Comment by codefromthecrypt** [comment]
TL;DR; this was implicitly rolled back into a second/review session of the existing time slot, then canceled. There's some interest in a 3 timezone proposal. We want UTC+8 to be able to attend in normal working hours, and 8am is acceptable. Let's bring it back as a meeting of peers, not a review session of decisions made by other regions.

---

What was proposed here eventually turned into a second meeting on the same day as the primary one, and eventually canceled. This seems in hindsight an unsurprising end as there's no incentive to join, as review meetings don't lead to impact or maintainer, etc (my opinion). It ends up a predictable road towards cancelation and this has happened outside CNCF also.

There's a resurgence of interest, but specifically that a meeting friendly to UTC+8 is a primary meeting, not a review meeting. That way, it doesn't end up being canceled from lack of purpose as we all can just watch videos. The idea is to better leverage timezones like this one, which have >20pct of the earth's population. We should be able to grow the community in a way that also doesn't require one part of the earth to be watchers, or have to miss sleep or family requirements.

Initially, this was a request to make two times, the existing unfriendly to APAC one and a second primary meeting 8am UTC+8. This is specifically 8am not 7am as existing contributors have family responsibilities. 8am is a good choice as it is extremely friendly to west coast US canada etc, at 4pm. This overlap is important to make the meeting sustainable. The agenda should be exclusive topics not simply rehash or study session. To get quorum we need to begin by folks sharing their influence, by presenting topics in the UTC+8 friendly time. Once more contributors grow there will both be a point proven (we desire to and did grow the community) and the region becomes more self-sufficient and equals with others. 

We *must* do everything we can to avoid the well worn track of setting this up for failure. If we only make this a study session of other timezones, or are impatient to quickly cancel, or folks in west coast don't attend at all, basically it is our fault, not the fault of the region that this failed. Let's not do that.

Later, there was interest in.. well, let's also recognize that if we do 50/50 2 meetings, it is very unbalanced vs today. Some people would like to attend more discussions and 8am UTC+8 is very unfriendly to europe. So, instead of them going from 100% content to 50%, to step down more gracefully to 2/3.. via a 3 meeting system. I believe this is an earnest interest in making sure this is viable and again not a path to canceling UTC+8 again. So, personally, sounds fine.

**Comment by andrzej-stencel** [comment]
Here's the link to the CNCF Slack conversation in #otel-collector channel started on 2024-06-26: https://cloud-native.slack.com/archives/C01N6P7KR6W/p1719363899132969.

**Comment by codefromthecrypt** [comment]
here's the community roll-up issue I was asked to create by @svrnm https://github.com/open-telemetry/community/issues/2176

**Comment by codefromthecrypt** [comment]
works for me. I think it may help with attendance to do a couple things, but up to you.
* date the slots based on the 9am, not normalize to california time (to avoid people thinking wrong day).
* be clear about this being rotation vs one long day, so one week, followed by the next, etc.
* [schedule](https://docs.google.com/document/d/15HSxJaUJojA1dPuLQdiVuUVGC6YsuPuunhb8TIFfRBs/edit) the first "slot 2" for two weeks from now, so that we can get the word out. So, 18 july UTC+8

wdyt?

**Comment by jpkrohling** [comment]
Sounds good. @open-telemetry/collector-approvers , @open-telemetry/collector-contrib-approvers , @open-telemetry/collector-maintainers , @open-telemetry/collector-contrib-maintainer , do you have any concerns, or do you think we can move forward with this?

**Comment by codefromthecrypt** [comment]
fyi the old meeting slot was still on the google calendar, so @andrzej-stencel @axw @mx-psi and I joined. A large part of this time was discussion engagement, how to build folks into roles, how to reduce pressure to have meetings, and encourage meetings in a way that will work (this time) cheers!

**Comment by mx-psi** [comment]
Expanding from @codefromthecrypt comment: we still have the EMEA-APAC meeting happening monthly at 9am CEST. Some notes about the experience there:
- Average attendance is about 1-2 people and (most of the time) @andrzej-stencel and me
- We frequently have to postpone decisions to the usual meeting time and rely stuff on people's behalf

The fact that it doesn't work great is not necessarily an argument against restoring the NASA-APAC meeting, we may have not publicized the new meeting times well enough and we may not have a sufficiently well-defined decision process, these are both things we can address

**Comment by codeboten** [comment]
Discussed in 03-Jul-2024 SIG call, it would be great to capture some details as to how decisions can be made through ad-hoc meetings when synchronous discussions are needed as maintainers/approvers may not be able to attend the split meetings.

I really like the idea of making this community more inclusive to other timezones. I'd also propose we run this as an experiment for a set amount of time (3/6 months) to validate that the new split meetings are working to continue to move the project forward. After which we can review community engagement to ensure it is continuing to foster the collaboration that is needed.

There was an attempt to do this in the Python community many years ago and we saw a drop in engagement because of the split. This is why I'm suggesting we schedule a time to review engagement.

**Comment by svrnm** [comment]
Thank you all for taking a look into this, it would be great if the larger community can profit from your experience after those 3-6 months as well, e.g. via https://github.com/open-telemetry/community/issues/2176

**Comment by jpkrohling** [comment]
Something else we discussed: I changed the original proposal (see my updated earlier comments), so that ET, Brazil, and India have two slots within the regular working hours, and APAC one. This is fair to our existing contributors on those time zones, while still being an improvement to the APAC folks.

**Comment by codefromthecrypt** [comment]
@jpkrohling you were discussing [this comment](https://github.com/open-telemetry/opentelemetry-collector/issues/7085#issuecomment-2199786190) and fwiw I agree it is a pragamatic step forward and in consideration of folks concern that a split will reduce decision throughput. 1/3 of meetings being possible for UTC+8 is still progress vs none, and we can work with that IMHO 👍 

nit: I might not call slot three EMEA/APAC anymore as I'm not sure that's the best way to phrase it as it isn't very much about apac anymore. maybe just call it EMEA as it is pretty well centered there?

**Comment by axw** [comment]
There seems to be an off-by-one (daylight savings?) in the NASA/APAC meeting scheduled for July 17/18:

![image](https://github.com/open-telemetry/opentelemetry-collector/assets/843579/4b51d729-bb52-4056-a874-1d68429090ee)

(I'm in Perth, same timezone as Beijing - UTC+8.)

**Comment by jpkrohling** [comment]
> There seems to be an off-by-one (daylight savings?) in the NASA/APAC meeting scheduled for July 17/18:

That's still the old entry, I'll work on a new entry once we have enough +1 from Collector folks, which I think we informally have already.

**Comment by jpkrohling** [comment]
There are two PRs for this proposal:

- Community PR, splitting the meeting times in three slots: https://github.com/open-telemetry/community/pull/2184
- Changes to the readme, addressing the concerns mentioned by @codeboten and discussed during the SIG meeting: #10536

**Comment by codefromthecrypt** [comment]
TL;DR;

The schedule is Wednesdays, rotating between the following time slots: 00:00 UTC, 12:00 UTC, 16:00 UTC

This means the next in APAC is Weds 24 July at 8am GMT+8
